measure.rb

```diff
@@ -829,15 +829,12 @@ def self.calc_approximate_view_factor(runner, model, all_surfaces)
              (s_tilt - s2_tilt).abs > same_ang_limit
             if surface2.subSurfaces.size != 0
               # calculate surface and its sub surfaces view factors
-              parent_surface_a = surface2.grossArea
+              parent_surface_a = surface2.netArea
+              if parent_surface_a > 0.01 # base surface of a sub surface: window/door etc.
+                surface_vf_map[surface2] = parent_surface_a / zone_seen_area
+              end
               surface2.subSurfaces.each do |sub_surface|
-                parent_surface_a -= sub_surface.grossArea
-                if parent_surface_a < 0.01 # base surface of a sub surface: window/door etc.
-                  surface_vf_map[sub_surface] = surface2.grossArea / zone_seen_area
```

Comment:
This line is not correct if it has more than one sub surfaces, I reverted sub surface view factor calculation the same as before.